api.planx.uk/modules/test/routes.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from "express";
+import { testSessionMethods } from "./controller";
+
+const router = Router();
+
+// in order to test the session setup, we need a dedicated test route
+router.get("/test-session", testSessionMethods);
+
+export default router;
```

Comment:
Seems like a solid light-touch solution to the `supertest` limitation - I like this approach !